src/wechaty/user/contact.py

```diff
@@ -72,6 +72,21 @@ def __init__(self, contact_id: str):
         # self.name: str = "default_acontact"
         self.payload: Optional[ContactPayload] = None
 
+    _event_stream: AsyncIOEventEmitter = AsyncIOEventEmitter()
```

Comment:
Are there any blockers that prevent we inherited Contact from the EventEmitter class?

Reply:
No, python class can inherit many base-class, It's all my code habits of writing other languages.
I will refactor them at later commit.

Reply:
At the beginning, I think that Contact/Room class should focus on logic processing. One module should be as simple as possible.  If we add EventEmit ability to them, they look redundant. Although, I have not split the code. Besides，Event listener should be global function, not a member of a class.

For example, `room-invite` event should be a moduler function, don't couple with Contact/Room class.

```python
# user/room.py
class Room:
    pass

# user/room_events.py
def listen_room_invite(room: Room, inviter: Contact, ):
    # do some logic processing
```

But, If we put all things which is about room to`Room` class, that also looks good to me.  If I make`Room` inherited from `AsyncIOEventEmitter `, there is only a few code changes. So, it's simple to refactor code.

What do you think about it.

Reply:
ic. Let's talk about it in today's meeting.